lib/seattleflu/id3c/cli/command/etl/redcap.py

```diff
@@ -611,10 +612,12 @@ def cast_to_integer(string: str) -> Optional[int]:
 
     def cast_to_float(string: str) -> Optional[float]:
         try:
-            return float(string)
+            n = float(string)
         except ValueError:
             return None
 
+        return n if isfinite(n) else None
```

Comment:
I debated whether to put this logic into this utility but didn't like the idea that the caller wouldn't know that it was happening. It's possible that the float value is expected to always have a legit value and the caller would want to know that it sometimes doesn't--for example, we might need to fix the REDCap calculation.

Reply:
Ah, hmm. For the future, that sort of reasoning and context is exactly the sort of thing I think is really valuable to include in commit messages (e.g. in e6a089d in this case).

I agree it's possible the caller might want to be able to know that a non-finite value was received… but that seems at odds with the current `cast_to_float` behaviour of converting all `ValueError`s to `None`.